notebooks/example_persontransportstatistik.py

```diff
@@ -0,0 +1,32 @@
+"""Example."""
+
+# %% [markdown]
+# ## Example on how to run
+
+# %% [markdown]
+# Import packages.
+
+# %%
+import json
+
+# %matplotlib inline
+import ifk_scb_compilations.objects.passenger_transport as passenger_transport
+import requests
+
+# %% [markdown]
+# Start a request session and fetch data from scb, based on the query defined in ifk_scb_compilations.queries.passenger_transport.Request_input.query.
+
+# %%
+session = requests.Session()
+response = session.post(
+    passenger_transport.Request_input.url, json=passenger_transport.Request_input.query
+)
+response_json = json.loads(response.content.decode("utf-8-sig"))
```

Comment:
Is the idea to move this into the package?

Reply:
I did so, and some small re-structure.